src/EFTools/DesignXmlCore/VisualStudio/NativeMethods.cs

```diff
@@ -901,9 +901,8 @@ public void SetSize()
         [DllImport("user32", ExactSpelling = true, CharSet = CharSet.Unicode)]
         public static extern short GetKeyState(int keyCode);
 
-        [SuppressMessage("Microsoft.Portability", "CA1901:PInvokeDeclarationsShouldBePortable")]
         [DllImport("user32", ExactSpelling = true, CharSet = CharSet.Unicode)]
-        public static extern int MsgWaitForMultipleObjects(int nCount, int pHandles, [MarshalAs(UnmanagedType.Bool)] bool fWaitAll, int dwMilliseconds, int dwWakeMask);
+        public static extern int MsgWaitForMultipleObjects(int nCount, IntPtr[] pHandles, [MarshalAs(UnmanagedType.Bool)] bool fWaitAll, int dwMilliseconds, int dwWakeMask);
```

Comment:
I'm not sure if you need to explicitly state MarshalAs(UnmanagedType.LPArray), or if that is the default for pInvoke. Could double-check with Aaron Robinson (not sure if he has a github or what it is to tag him here), just to ensure it isn't accidentally working somehow without said attribution (or you could add it just to be explicit in the intention (i.e. what the Win32 API itself is expecting, which is LPArray)).

Reply:
I think LPArray is the default for P/Invoke. I thought about annotating and also including the count linking to the other parameter but that's only needed for interop in the other direction here.